packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -135,11 +163,19 @@ export default {
             return this.isFieldEmpty(VALIDATIONS.address, 'postcode');
         },
 
+        isAddressAdministrativeAreaEmpty () {
+            return this.isFieldEmpty(VALIDATIONS.address, 'administrativeArea');
+        },
+
         /*
         * Checks that postcode is a valid postcode and that the field is not empty.
         */
         isAddressPostcodeValid () {
             return (!this.$v[VALIDATIONS.address].postcode.$dirty || this.$v[VALIDATIONS.address].postcode.isValidPostcode) && !this.isAddressPostcodeEmpty;
+        },
+
+        shouldShowAdministrativeArea () {
```

Comment:
Is it work moving this into checkout and passing it through as a prop. Then the validations bit [here](https://github.com/justeat/fozzie-components/pull/1245/files#diff-3f7bda367e6ae2f5c141559c1aec5722d6606ee4d895f7d44e9837c20f7efb3cR1140) can use it too?

Reply:
Yeah, nothing wrong with that. I'll update.